pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+## Fixes Issue
+
+<!-- Example: Closes #61 -->
+
+## Changes proposed
+
+<!-- List all the proposed changes in your PR -->
+
+
+## Screenshots
```

Comment:
Don't think we need a dedicated Screenshots section, as screenshots can be included as motivated to explain any proposed changes

Reply:
How about I add the tag "Optional"?

Reply:
Sounds reasonable, though could be cleaner to specify this in a comment within the `Changes proposed` section. Either works

Reply:
@nadir-akhtar-coinbase made some changes

Thoughts??